### ISSUE TITLE
Labels on data points in 3D scatter plots?

### ISSUE THREAD

**Issue by deeplook** [maartenbreddels/ipyvolume#115]
I don't see a way of adding text labels on individual data points in a 3D scatter plot. Is this a future feature, maybe?

**Comment by maartenbreddels** [comment]
There is no way yet, and yes it should happen! :)

**Comment by nanodust** [comment]
is there a way to do 'picking' meanwhile ? ie retrieve ID of object under cursor ?

**Comment by maartenbreddels** [comment]
No, not yet, please open a separate issue for this, so I can keep track of it more easily. Happy to discuss it further!

**Comment by robsalasco** [comment]
+1 I would like to see this feature!

**Comment by alc5978** [comment]
+1 I would like to see this feature!

Many Thanks for your work

**Comment by eshernan** [comment]
+1 I would like to see this feature

**Comment by EmmanuelArbogast** [comment]
+1 I would like to see this feature

**Comment by miltonArango** [comment]
+1 I would like to see this feature

**Comment by robsalasco** [comment]
I like how the embedding projector solves the problem of labelling. Would be so nice to have something like that https://projector.tensorflow.org
